packages/api/lib/writeRecords/write-granules.js

```diff
@@ -463,7 +463,7 @@ const _writeGranule = async ({
  * @param {Object} [params.granulePgModel] - only for testing.
  * @param {Knex} knex - knex Client
  * @param {Object} esClient - Elasticsearch client
- * @param {string} snsEventType - SNS Event Type
+ * @param {string} snsEventType - SNS Event Type, defaults to 'Create'
```

Comment:
This is no longer accurate

Reply:
[`a18bcde` (#2592)](https://github.com/nasa/cumulus/pull/2592/commits/a18bcde9747312de1761d9f2ba649978d9570dc4)